wagtail/signal_handlers.py

```diff
@@ -18,13 +18,14 @@
 logger = logging.getLogger("wagtail")
 
 
-# Clear the wagtail_site_root_paths from the cache whenever Site records are updated.
 def post_save_site_signal_handler(instance, update_fields=None, **kwargs):
-    Site.clear_site_root_paths_cache()
+    # NOTE: applies to the current instance only
```

Comment:
Question: Current instance of what?

Reply:
I'm open to ideas on the most appropriate language to use here, but what I mean here is the 'current running instance' - in situations where the project had multiple running instances (most likely in separate docker containers)

Reply:
Decided to remove these comments to avoid ambiguity. I think the docstrings on the methods themselves explain things well enough.

Reply:
Resolved in [601112e](https://github.com/wagtail/wagtail/pull/10018/commits/601112eebe944813e4abb1ed691f83af97062469)